packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -106,29 +106,45 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
   private _initializeApp() {
     // TODO: Maybe pass this in as a parameter instead of creating it here?
     this.state.app = express()
+    if (this.options.ethNetworkName) this._initMonitoring()
+    this.state.app.use(cors())
+    this._registerAllRoutes()
+    this.state.app.use(Sentry.Handlers.errorHandler())
```

Comment:
What happens if sentry is not initialized and this is added to the middleware? Perhaps this should happen inside of `_initMonitoring`

Reply:
If you want it to be last, then maybe it should be behind an if statement?

Reply:
ooo this is a great call out, i missed this line during the copy pasta so will fix